docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,46 @@
+.. _config_http_filters_cache:
+
+HTTP Cache Filter
+=================
+**Work in Progress--not ready for deployment**
+
+HTTP caching can improve system throughput and latency and reduce network and
```

Comment:
Nit: "throughput, latency and reduce network/backend loads.." (too many ands)

Reply:
done